app/views/questions/_answer.html.erb

```diff
@@ -14,6 +14,9 @@
         <a style="color:#aaa;" href="#a<%= answer.id %>"><%= time_ago_in_words(answer.created_at) %> ago</a>
       </div>
       <div class="navbar-text pull-right">
+        <a rel="tooltip" title="Flag as spam" class="btn btn-sm btn-default btn-flag-spam-<%= comment.id %>" href="mailto:moderators@publiclab.org?subject=Reporting+spam+on+Public+Lab&body=Hi,+I+found+this+comment+that+looks+like+spam+or+needs+to+be+moderated:+https://publiclab.org/<%= comment.parent.path %>#c<%= comment.cid %>+by+https://publiclab.org/profile/<%= comment.author.username %>+Thanks!">
```

Comment:
Ah! let's change that to `answer.id` -- sorry, missed that!!

Reply:
Just the very first ```<%= comment.id %>``` to ```<%= answer.id %>```  and leave the ```<%= comment.parent.path %>#c<%= comment.cid %>+by+https://publiclab.org/profile/<%= comment.author.username %>+Thanks!">``` as is?